kogito-editors-js/packages/import-java-classes-component/src/__tests__/ImportJavaClasses/ImportJavaClasses.test.tsx

```diff
@@ -99,9 +98,83 @@ describe("ImportJavaClasses component tests", () => {
     expect(baseElement).toMatchSnapshot();
   });
 
+  test("Should move to second step", async () => {
```

Comment:
It seems we are awaiting anything. Is this `async` really needed?
Also, a suggestion from a readability perspective: maybe you could split the code of this test in meaningful methods

Reply:
@vpellegrino I applied your suggestions. Thx u!